spec/hashie/mash_spec.rb

```diff
@@ -137,9 +137,9 @@
 
   include_context 'with a logger' do
     it 'logs a warning when overriding built-in methods' do
-      Hashie::Mash.new('trust' => { 'two' => 2 })
+      Hashie::Mash.new('object_id' => { 'two' => 2 })
```

Comment:
**praise:** This makes sense and is not going to change in the future so is a good choice!